scormxblock/scormxblock.py

```diff
@@ -289,6 +293,7 @@ def publish_grade(self):
                     'value': self.lesson_score,
                     'max_value': self.weight,
                 })
+        self.publish_completion()
```

Comment:
Completion status of xblock should not depend on whether it is scorable or not. I think at the end of `scorm_set_value` method where we are pulling completion status we should add these lines to mark completion
```
completion_status = self.get_completion_status()
if completion_status in ["passed", "failed", "completed"]:
    self. emit_completion(1.0)
```